draft-fdb-rats-psa-endorsements.md

```diff
@@ -84,39 +89,47 @@ cryptographic key material needed to verify Evidence signed by the device's PSA
 RoT. Additionally, PSA Endorsements can include information related to the
 certification status of the attesting device.
 
-There are five types of PSA Endorsements:
+There are three basic types of PSA Endorsements:
```

Comment:
As we have removed "Block List Triples" from here it is better we remove it from everywhere in the document. The last section on Block List is quite confusing!

Reply:
ok